DQMOffline/Trigger/python/NoBPTXMonitor_cff.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.NoBPTXMonitor_cfi import hltNoBPTXmonitoring
+
+hltNoBPTXMuonsMonitoring = hltNoBPTXmonitoring.clone()
+hltNoBPTXMuonsMonitoring.FolderName = cms.string('HLT/NoBPTX/Muons/')
+hltNoBPTXMuonsMonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_L2Mu40_NoVertex_3Sta_NoBPTX3BX_v*") #HLT_ZeroBias_v*
```

Comment:
Sorry I didn't see this comment and your one below until now, Mia. My latest commit should add the two backup paths.